tests/test_dataset.py

```diff
@@ -48,7 +48,8 @@ def test_shape(self, expected_shape):
                 'extra': os.path.join(tempdir, 'test_extra2.nii.gz')
             }
         ]
-        dataset = Dataset(data=test_data, transform=LoadNiftid(keys=['image', 'label', 'extra']))
+        test_transform = Compose([LoadNiftid(keys=['image', 'label', 'extra']), TestIdentity(minimum_time_delay=1e-7)])
```

Comment:
Could you please help create 2 new unit tests for `TestIdentity` and `TestIdentityd` directly?
Thanks.